src/consent.js

```diff
@@ -78,6 +78,23 @@ export function getConsentPolicyInfo(element, policyId) {
   );
 }
 
+/**
+ * @param {!Element|!ShadowRoot} element
+ * @param {string} policyId
+ * @return {!Promise<boolean>}
```

Comment:
nit: {!Promise<?boolean>}

Developers need to be very careful about `null` vs `false` here.

Reply:
Oops my b